controllers/managedocs_controller_test.go

```diff
@@ -1021,6 +1072,65 @@ var _ = Describe("ManagedOCS controller", func() {
 				}, timeout, interval).Should(Equal(ctrlutils.GetResourceRequirements("ocs-operator")))
 			})
 		})
+		When("there is a notification email address in the add-on parameter secret and smtp secret", func() {
+			It("should update alertmanager email config with smtp details", func() {
+				secret := addonParamsSecretTemplate.DeepCopy()
+				secretKey := utils.GetResourceKey(secret)
+				Expect(k8sClient.Get(ctx, secretKey, secret)).Should(Succeed())
+
+				// Add notification email to the addon param secret
+				secret.Data["notification-email-0"] = []byte("test@email.com")
```

Comment:
This is another candidate for param to the `setupAlertmanagerConfigConditions` util I suggested in the last comment. You can even use an array to set up multiple addresses at once.